stylesheet.css

```diff
@@ -0,0 +1,275 @@
+
+nav{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+    background-color: #d1e0ef;
+    text-transform: uppercase;
+}
+.topnav{
+    padding: 12px;
+}
+nav a{
+    margin: 10px;
+}
+nav a:hover{
+    background-color:#fae3e5;
+    border-radius: 25px;
+    padding: 12px;
+}
+a:hover{
+    color: #e2431b;
+}
+/* Header styling  */
+header{
+    background-image: url("Images/Background.jpg");
+    height: 750px;
+    text-align: center;
+}
+.intro{
+    display: grid;
+    grid-template-columns: 60% 40%;
+    padding-top: 100px;    
+}
+.intro_item{
+    margin: 30px;
+    align-self: center;
+}
+.intro_item.intro_text{
+    background-color: #FFFFFF ;
+    padding: 100px 40px;
+    margin-left: 150px;
+    text-align: left;
+}
+.profile{
+    object-fit: cover;
+    border-radius: 50%;
+    height: 300px;
+    width: 300px;
+    display: block;
+    margin: auto;
+}
+p{
+    padding-bottom:10px;
+}
+/* header button styling */
+.header_button{
+    display: inline-block;
+    font-size: 20px;
+    border-radius: 45px;
+    border-style: none;
+    background-color: #FFFFFF;
+    padding: 30px;
+}
+.jump_to_contact{
+    text-decoration: none;
+}
+/* My work section styling */
+.my_work{
+    display: grid;
+    grid-template-columns: 40% 60%;
+    padding-top: 100px;
+    background-color: #d1e0ef; 
+}
+.project_text{
+    padding: 40px;
+    margin-left: 50px;
+}
+.project_image img{
+   height: 400px;
+   padding: 30px;
+   display: block;
+   margin-left: auto;
+   margin-right: auto;
+}
+.my_work_two{
+    display: grid;
+    grid-template-columns: 60% 40%;
+    padding-top: 100px;
+    background-color: #d1e0ef;
+}
+.external_button:hover{
+    background-color: #fae3e5;
+}
+
+.external_button{
+    padding:18px;
+    background-color: #FFFFFF;
+    border-radius: 45px;
+    margin: 10px 10px 0 0;
+    display:inline-block;
+
+}
+/* My experience section styling */
+.my_experience{
+    background-color: #fae3e5;
+    margin: 0px;
+    display: grid;
+    grid-template-columns: 40% 30% 20%;
+}
+.cv_image{
+    max-width: 500px;
+}
+.cv_icon{
+    place-self: center;
+}
+.experience_text{
+    padding: 100px;
+}
+/* Contact form styling */
+.contact_form_section{
+    background-image: url("Images/Background.jpg");
+    height: 850px;
+    padding-top: 100px;
+}
+.contact_form{
+    background-color: #FFFFFF;
+    padding: 100px;
+    margin: auto;
+    width: 50%;
+}
+.contact_form p{
+    margin-left: 10px;
+}
+.contact_form h2{
+    margin: 10px;
+}
+input{
+    display: block;
+    padding: 10px;
+    margin: 10px;
+    background-color: #d1e0ef;
+    border: none;
+    width: 90%;
+}
+textarea{
+    margin: 0 10px;
+    padding: 1px 2px;
+    background-color: #d1e0ef;
+    border: none;
+    padding: 10px;
+    color: #000000;
+    width: 90%;
+}
+textarea::placeholder{
+    color: #6a6a6a;
+}
+/* Footer bar styling */
+footer{
+    height: 70px;
```

Comment:
Would suggest not setting a height here, and instead using the padding to get the height you want